index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!--Homepage view-->
+    <section class="homepage">
+        <!--header and headline sections-->
+        <section class="header">
+            <!--header section-->
+            <nav class="navigation_bar">
+                <span class="my_logo">My Logo</span>
+                <img class="social_media_icons" src="icons/Union.png" alt="">
+            </nav>
+        </section>
+        <!--headline section-->
+        <section class="headline">
+            <div class="headline-1">
+                <h1 class="headline-1-h1">I’m Mariane
+                    <br>Glad to see
+                    <br>you!
+                </h1>
+            </div>
+            <div class="headline-2">
+                <p class="headline-2-p">I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience!
+                     If you like what you see and have a project you need coded, don’t 
+                     hestiate to contact me.
+                </p>
+            </div>
+            <div class="headline-3">
+                <p class="headline-3-p">LET’S CONNECT</p>
+            </div>
+            <div class="headline-4">
+                <img class="headline-4-img" src="icons/social media icons.png" alt="">
+            </div>
```

Comment:
- [ ] Please I have noticed that you use an image instead of using the `ul` and `li` tags to list the social media icons in your project. Make sure that you should use the `a` tag to wrap each icon `img` tag in your project. So that when you click on the social media icon it will take you to the social media page. :pray: